apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-site-editor-sidebar-clicks.ts

```diff
@@ -39,20 +40,28 @@ export const wpcomSiteEditorSidebarPatternsClick = () => {
 	};
 };
 
-export const wpcomSiteEditorSidebarStylesClick = () => {
+export const wpcomSiteEditorSidebarStylesClick = (): DelegateEventHandler => {
 	return {
 		id: 'wpcom_site_editor_sidebar_styles_click',
-		// \2f is the encoded slash.
-		selector: '#\\2fwp_global_styles',
+		/**
+		 * There is no id attribute (`#/wp_global_styles`) for themes that do not have style variations.
+		 * Make this selector generic so as to not depend on the DOM structure.
+		 */
+		selector: '.edit-site-sidebar-navigation-item',
 		type: 'click',
-		handler: () =>
+		handler: ( event ) => {
+			const target = event.target as HTMLElement | null;
+			if ( target?.textContent !== 'Styles' ) {
```

Comment:
Unfortunately, we cannot depend on the text because of translations. 😞

Reply:
Updated: https://github.com/Automattic/wp-calypso/pull/81002/commits/7de7a28b0de7dd7b38e19280caa7222b2a5d08bd